src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -302,10 +302,18 @@ const AccountClosure = (() => {
             collapsible: true,
             active     : true,
         });
+        const accountclose_dialog = $('#account-closure-dialog');
```

Comment:
please use our jquery naming convention,
```suggestion
        const $account_closure_dialog = $('#account-closure-dialog');
```